.github/workflows/publish_website.yml

```diff
@@ -0,0 +1,62 @@
+name: Publish Website
+
+on:
+  workflow_call:
+    inputs:
+      new_version:
+        required: false
+        type: string
+      run_tutorials:
+        required: false
+        type: boolean
+        default: false
+  workflow_dispatch:
+  push:
+    branches: [ main ]
```

Comment:
Do we want to publish the website on each push to the main branch? Leftover from testing?

Reply:
You're right this is leftover from testing. Removing

Reply:
Removed in [3e6d0b6](https://github.com/pytorch/botorch/pull/2653/commits/3e6d0b6ca6cd31e737bd25d0e0a1771f4f583474)